src/main/java/seedu/tasklist/logic/commands/ListCommand.java

```diff
@@ -40,7 +40,7 @@ public ListCommand(Index numberOfDays) {
 
     private Predicate<Assignment> showLimitedAssignments() {
         return assignment -> {
-            DateTimeFormatter inputFormat = DateTimeFormatter.ofPattern(Deadline.DEADLINE_DATE_TIME_FORMAT)
+            DateTimeFormatter inputFormat = DateTimeFormatter.ofPattern(Time.DEADLINE_DATE_TIME_FORMAT)
```

Comment:
Maybe it would be good to rename the final static variable to `TIME_DATE_TIME_FORMAT` as well as I believe the Lesson class also makes use of the same variable to format the start and end times